### ISSUE TITLE
Need to know : how to use this script

### ISSUE THREAD

**Issue by HarshPrajapati2309** [ocrmypdf/OCRmyPDF-EasyOCR#11]
I am trying to use this script but couldnt find the way

**Comment by jbarlow83** [comment]
Did you follow the instructions in the README?
If that succeeds, ocrmypdf and this plugin will be installed.

**Comment by sliedes** [comment]
I think the question may be about how to use this with OCRmyPDF; i.e. yes, it's installable, but how to run OCRmyPDF with it without running into `NotImplementedError: EasyOCR does not support hOCR output`.

And I suspect the answer may be to use the OCRmyPDF option `--pdf-renderer sandwich`?

**Comment by jbarlow83** [comment]
Yes, you need to use `--pdf-renderer sandwich`.

**Comment by sliedes** [comment]
BTW, I made a PR to EasyOCR that adds hOCR output (though didn't test it with this plugin): https://github.com/JaidedAI/EasyOCR/pull/1275

**Comment by jbarlow83** [comment]
@sliedes Can you contribute your hOCR change here since EasyOCR doesn't seem to merge many PRs?

**Comment by sliedes** [comment]
> @sliedes Can you contribute your hOCR change here since EasyOCR doesn't seem to merge many PRs?

Certainly! Do you need me to open the PR here, or can you just pick it from the EasyOCR repository?

**Comment by jbarlow83** [comment]
Please open a PR here - that would be great, thank you!

**Comment by sanjay-nit** [comment]
Even after using the `--pdf-renderer sandwich` option, when I copy text from the converted PDF and paste it somewhere, it outputs garbled text. 
```
 €(.|'; 0 #^| ?|*)+ 0 "^| 6&: 9@ ,/.$_-0'; €, {6 +//:'6+. 6$|6( &.+, > &/&< ,//|-"$  60) 0 ,|*)+$ ') +| '|*)+'$ €'"-'*0); && +0/06* +0 #^| €"+.:) '*&6:*, $; *$*$; :€*..)9 *^:*..* €"-3/:' +|&*€)+ 0/|$< 6( *^| +|)*,
```

**Comment by jbarlow83** [comment]
> Even after using the `--pdf-renderer sandwich` option, when I copy text from the converted PDF and paste it somewhere, it outputs garbled text. 

Open a new issue, provide your input file, command line arguments, and the program you are using to extract text. 

Otherwise is it not possible to work out what is going wrong.

**Comment by alaminkouser** [comment]
Set language

```python
ocrmypdf.ocr(os.path.join(save_path, "input.pdf"), os.path.join(save_path, "output.pdf"), force_ocr=True, pdf_renderer="sandwich", language="eng")
```
